### PR TITLE
added activity selection problem

### LINKED ISSUE #488: adding greedy algorithms in python
**Is your feature request related to a problem? Please describe.**
Hey @rcoder23  I would like to add all major greedy  algorithms like Fractional Knapsack
Huffman Coding, Activity Selection Problem, Egyptian Fraction in python language

**Do you want to work on it**
Yes
I will implement this in python language

Kindly Assign me this under GSSOC 21
Thank you

### PR DESCRIPTION
**Pull request by robin025**
hey @rcoder23 
fixes #488 
added activity seclection problem

### CODE REVIEW

File: greedy algorithms/Activity_Selection_problem.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                start[i],finish[i] = a,b
+            activity_selection_prob(start, finish, len(start))
+
+
+
+
+
+
+
+
```
Comment (rcoder23) [inline greedy algorithms/Activity_Selection_problem.py:43]: remove spaces

File: greedy algorithms/Activity_Selection_problem.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+# Start ?1                                                                                                                finish ?9                                                                                                               Entre Start value And finish Value                                                                                      
+# Start ?5                                                                                                                finish ?8                                                                                                               Following Activities Are selected :                                                                                     
+# Index   Start Finish                                                                                                  0             5-->9                                                                                                
+# 1.Entre  Activity       2.Exit
```
Comment (rcoder23) [inline greedy algorithms/Activity_Selection_problem.py:57]: Use """       """    for comments

**Review by rcoder23 — CHANGES_REQUESTED** [review]
Add solution with explaination

File: greedy algorithms/Activity_Selection_problem.py (hunk 1 above)
Comment (robin025) [reply greedy algorithms/Activity_Selection_problem.py:43]: okay @rcoder23

File: greedy algorithms/Activity_Selection_problem.py (hunk 2 above)
Comment (robin025) [reply greedy algorithms/Activity_Selection_problem.py:57]: ok @rcoder23

File: greedy algorithms/Activity_Selection_problem.py (hunk 1 above)
Comment (robin025) [reply greedy algorithms/Activity_Selection_problem.py]: Done @rcoder23

File: greedy algorithms/Activity_Selection_problem.py (hunk 2 above)
Comment (robin025) [reply greedy algorithms/Activity_Selection_problem.py]: Done @rcoder23

**Comment by robin025** [comment]
@rcoder23  maded changes as suggested

**Review by rcoder23 — APPROVED** [review]

**Comment by robin025** [comment]
Hello @rcoder23 ,
I had a discussion with Praveen who is doing all the leaderboard stuff of GSSoC21 and he told all of us that the GSSoC21 tag should look like *GSSoC21* and it's case sensitive, so if the tag doesn't look like this *GSSoC21* then that PR will not be counted, so can you please update the tag in all my PRs thatbhave reference to issue #488 
Note : there are only 4 PRs

Thankyou

**Pull request merged**